src/test/app/Escrow_test.cpp

```diff
@@ -960,10 +1193,8 @@ struct Escrow_test : public beast::unit_test::suite
         env.memoize("carol");
 
         {
-            auto const jtx = env.jt(
-                condpay("alice", "carol", XRP(1000),
-                    makeSlice(cb1), env.now() + 1s),
-                seq(1), fee(10));
+            auto const jtx = env.jt(escrow("alice", "carol", XRP(1000)),
+                finish_time(env.now() + 1s), seq(1), fee(10));
```

Comment:
This may have been intentional, but the revised test doesn't have the same effect as the old test (it's not conditional).  Just checking because that doesn't match the pattern of most of the rest of the unit test changes.

Reply:
The conditional wasn't really useful, so I ditched it, because by this point in the changes I was getting tired :stuck_out_tongue_closed_eyes: